.github/workflows/ci.yml

```diff
@@ -47,6 +47,11 @@ jobs:
             rust-target: aarch64-unknown-linux-musl
             binary: fluvio
             run: make build-cli
+          - os: ubuntu-latest
+            rust: stable
+            rust-target: arm-unknown-linux-gnueabihf
+            binary: fluvio
+            run: make build-cli
```

Comment:
don't add run

Reply:
This entire job is structured this way, I am just adding one include rule

![image](https://user-images.githubusercontent.com/4210949/127196100-bda55175-9d9d-46b2-a7d5-c6fe8228103c.png)

As I mentioned above, I don't want to rewrite this job as part of this PR